<!--
F. Caballero and F.J. Perez-Grau
Files tuned for EuRoC dataset 'catec-2016-01-27-12-30-41.bag'
-->
<launch>

  <node name="viodom_node" type="viodom_node" pkg="viodom" output="screen" >
    <param name="left_cam" value="/multisense/left" />
    <param name="right_cam" value="/multisense/right" />
    <param name="imu_topic" value="/multisense/imu/imu_data" />
    <param name="downsampling" value="2" />
    <param name="publish_pc" value="true" />
    <param name="max_features" value="800" />
    <param name="flow_threshold" value="10.0" />
    <param name="min_matches" value="20" />
    <param name="src_frame_id" value="world" />
    <param name="tgt_frame_id" value="base_link" />
    <param name="display_images" value="false" />
  </node>

</launch>
